### ISSUE TITLE
Run the confirmatory oracle-symbol joint-controller experiment

### ISSUE THREAD

**Issue by Sino-Huang** [Sino-Huang/NovPhy#15]
## Parent

#1 — BG-NS-JEPA model and experimental design

## What to build

Run the untouched held-out-scenario experiment that accepts or does not support the central joint-control hypothesis under the predeclared matched-compute endpoint-gain and physical-violation rule.

## Acceptance criteria

- The strongest comparator is selected from the declared baseline suite before final test metrics are viewed.
- Compute budgets, practical-effect threshold, and endpoint physical-violation margin are fixed from pilot evidence before the confirmatory run.
- The learned joint controller is evaluated on held-out complete scenarios under the same physics family and ontology.
- If no declared compute budget passes the constrained-gain rule, report `not_supported_by_this_experiment`: a bounded result for the frozen candidate and protocol, not a general impossibility or ineffectiveness claim.

## Blocked by

- #2 — Placeholder: Make enriched BG-NS-JEPA training data available
- #9 — Establish fixed and independently optimized policy baselines
- #11 — Adapt the controller to closed-loop carrier states
- #12 — Test model-relative micro-constraint reliability
- #13 — Select the simplest effective symbolic interface
- #14 — Diagnose shared-predictor interference with pair experts
- #34 — Freeze the prospective statistical analysis protocol

**Comment by Sino-Huang** [comment]
Final-evaluation boundary handoff:

The public v5 release contains 18 non-final rollouts. Six final-evaluation rollouts are in sealed bundle `issue-53-final-evaluation-sealed-bundle-v5:mixed-termination`, bound to workflow `central-v2-final-evaluation-workflow-v5:issue-53`. The existing authorized issue-54 probe returned only an access receipt; it exposed no sealed rollouts, labels, observations, examples, or metrics.

#15 must continue to treat final outcomes as untouched. Actual access should occur only after #34 freezes the protocol, comparator, budgets, thresholds, seeds, analysis, and consumer authority, and should be recorded through the declared final workflow rather than `CohortV2ReleaseReader`, which rejects final-evaluation access.

**Comment by Sino-Huang** [comment]
Protocol-freeze handoff: #34 is closed as completed. Use protocol artifact cohort-v2-prospective-statistical-protocol-v1:sha256:4a189a5583c9bedf4fd5f670f14fd7ae2d4339c3a2beba95a1c5e269b02834e4 at data/runtime_evidence/issue-34/cohort-v2-prospective-statistical-protocol-v1.json, committed in 8e9544498456f7a629baebf22876d762e959349c. The six final attempt identities, two budget-specific comparators, thresholds, margins, seeds, paired bootstrap, multiplicity rule, failure handling, required outputs, and supported/unsupported decision are frozen. The sealed bundle remains unopened and the plan-v5 workflow remains pending; #15 must create the declared authorized derivative and access audit before opening final data. No final outcome may alter the protocol.

**Comment by Sino-Huang** [comment]
Issue #15 implementation is ready and pushed on env at 07b11fcfdb7bc514033ceffcfba9e0495dcbe92f.

The runner consumes the frozen #34 protocol and frozen #58 predictor/controller checkpoints, creates the declared issue-15 authorized workflow derivative before first final access, evaluates exactly the six sealed attempt identities, applies the two preselected budget comparators and 10,000-sample paired bootstrap rule, reports local teacher-forced metrics separately from fixed-h15 recursive diagnostics, and emits immutable evidence plus a compact summary. The ordinary reader remains unable to access final data.

Verification completed without opening final outcomes:
- 51 focused tests passed across confirmatory analysis, integrated model, controllers, aggregation, exhaustive evaluation, trajectory labels, and final access.
- Audit-order test confirms a pending workflow is rejected before sealed validation.
- No-write dry-run passed against the frozen #58 checkpoint.
- git diff --check passed; tracked worktree is clean and synchronized with origin/env.

Foreground production command (unbuffered progress logs):

    python -u -m scripts.run_cohort_v2_confirmatory --implementation-commit 07b11fcfdb7bc514033ceffcfba9e0495dcbe92f

After it completes, validate without reopening sealed source data:

    python -u -m scripts.run_cohort_v2_confirmatory --validate

The experiment is intentionally not run or the issue closed in this handoff because the real six-rollout exhaustive GPU evaluation is the long step delegated to the operator. No final outcome has been exposed by the implementation or dry-run.

**Comment by Sino-Huang** [comment]
## Confirmatory disposition: unsupported

Issue #15 is complete under the frozen #34 failed-run and falsification rules.

The authorized run opened exactly the six frozen final-evaluation attempts (1,968 frame records) under the declared workflow and recorded one passing access audit. Before any final metric or decision was produced, the exhaustive scorer failed in the source-bound oracle carrier codec: every final rollout contains 15 causal entities in every retained frame, while the issue-58 checkpoint declares 12 entity slots. Public training/calibration/model-selection rollouts contain at most 12 entities.

This is a candidate model execution failure, not an evaluator/GPU failure and not negative endpoint or physical-violation measurements. All six candidate replicates are retained as failed without replacement, exclusion, imputation, truncation, or outcome-conditioned rerun. Consequently:

- budget 106659.08223346318: candidate failed on 6/6; matched-capacity two-head comparison unavailable; budget rule failed;
- budget 1398656.0: candidate failed on 6/6; fixed-pair comparison unavailable; budget rule failed;
- paired bootstrap, fixed-h15 recursion, endpoint gain, and violation increase are unavailable because the candidate never reached valid evaluation;
- no declared budget passes, so the central joint-control hypothesis is unsupported.

Silently truncating to 12 entities would change the frozen codec semantics and checkpoint provenance after final access. A capacity-correct retraining/protocol would require a new model freeze and a new untouched final partition; it cannot rescue this confirmatory result.

### Evidence

- original implementation/run revision: 07b11fcfdb7bc514033ceffcfba9e0495dcbe92f
- failure-handling fix: b1dab5f79a2cadd35d0654c02fec88d2cd6d39c7
- compact-report fix: fa69f4f
- committed disposition evidence: 80e0028202f5931f4acab41db3fae800336b05b2
- compact report: data/runtime_evidence/issue-15/cohort-v2-oracle-symbol-confirmatory-summary.json
- full immutable evidence remains under .local-artifacts/issue-15-confirmatory

Verification: 54 focused tests passed; the no-write dry-run passed; failure-only evidence recomputes exactly; python -u -m scripts.run_cohort_v2_confirmatory --validate passed; git diff --check passed; env is clean and synchronized with origin/env.

This closes #15 as a completed unsupported disposition, not as positive evidence.

**Comment by Sino-Huang** [comment]
Corrective scientific interpretation and reopening:

The prior mechanical disposition followed the frozen #34 failed-run rule, but it did not produce endpoint, violation, bootstrap, controller, or recursive-rollout measurements. It therefore does not constitute evidence that the joint-control mechanism is ineffective. The accurate scientific status is inconclusive/not evaluable because the frozen issue-58 carrier had 12 entity slots while every opened final rollout had 15 causal entities.

Issue #15 is reopened because its intended confirmatory experiment remains unfinished. The opened six-rollout partition is permanently retired from confirmatory use and retained only as auditable failed-attempt provenance. Completion now requires: a capacity-compatible candidate trained and recalibrated using non-final roles only; a prospectively frozen versioned amended protocol; a newly generated untouched final partition with a separate authority and access audit; and the actual endpoint/physical-violation comparison on that new partition. No metric or structural detail from the opened partition may influence model selection, thresholds, margins, seeds, or the new partition design beyond the already established minimum representational requirement of at least 15 declared causal-entity slots.

**Comment by Sino-Huang** [comment]
## Capacity-correct confirmatory disposition: not supported by this experiment

Issue #15 is now scientifically complete under the prospectively frozen amended protocol v2. This result replaces the interpretation of the retired capacity-invalid run; it does not reuse or rescue the opened seed-4504 partition.

### Prospective correction and exposure audit

- The capacity-compatible integrated candidate was trained and recalibrated using only training, calibration, and model-selection roles: 15 entity slots, latent dimension 197, seed 20260824, 1,800 updates, batch 32, learning rate 3e-4, symbolic weight 1.0, controller seed 10, and one aggregation round.
- Calibration evidence was committed before the replacement final collection (`8fc45a9`, `037f508`).
- Protocol v2 and its six seed-4505 attempt identities were frozen and published before collection (`fca147d`, `219f82f`), with the bounded negative disposition `not_supported_by_this_experiment`.
- The new seed-4505 final partition produced exactly six accepted rollouts, 1,610 frame records, maximum 14 causal entities, one passing access audit, zero retries/replacements, and no reuse of seed-4504 (`7317bb8`, `e9f657a`).
- The capacity-valid evaluator and release-boundary correction were committed before the successful computation (`bc3e035`, `a504ccb`). The initial attempt under `bc3e035` stopped at a hard-coded old-release identity before writing any evaluation artifact; it did not inspect metrics or alter the frozen analysis.

### Frozen comparison result

All six paired complete rollouts were present at both declared budgets. There were no failed, missing, replaced, or excluded runs. Both candidate and comparator were within matched-compute support.

- Budget `106950.40762764572`, comparator `matched_capacity_two_head`: mean endpoint gain `0.0`; 95% bootstrap interval `[0.0, 0.0]`; one-sided 97.5% lower bound `0.0`, below the frozen practical threshold `0.000046003593412551444`. The physical-violation increase was `0.0`, with upper bound `0.0` at margin `0.0`. The violation rule passed; the gain rule did not.
- Budget `1402496.0`, comparator `fixed_pair`: mean endpoint gain `-0.000520526369465027`; 95% bootstrap interval `[-0.0007090821857106275, -0.00036250994663344503]`; one-sided 97.5% lower bound `-0.0007090821857106275`, below the frozen practical threshold `0.00016669932074791957`. The physical-violation increase was `0.0`, with upper bound `0.0` at margin `0.0003768522753780134`. The violation rule passed; the gain rule did not.
- Fixed-h15 complete-rollout diagnostic: six rollouts, mean terminal MSE `3.6777294079462686`, mean error AUC `165.9242770683874`, mean total compute `25478677.333333332`. Recursive predicted-carrier physical-violation labels remain unavailable by protocol.

No budget passed the frozen constrained endpoint-gain rule. The correct conclusion is therefore **`not_supported_by_this_experiment`**. This is bounded negative evidence for this candidate, partition, compute support, and decision rule—not a claim that joint control is generally ineffective or impossible.

### Evidence and verification

- result implementation: `a504ccb695fa583eea5bef5b85ef7528d5a33f87`
- committed compact evidence: `0868331`
- compact report: `data/runtime_evidence/issue-15/cohort-v2-oracle-symbol-confirmatory-v2-summary.json`
- full immutable evidence: `.local-artifacts/issue-15-confirmatory-v2`
- sealed replacement final bundle: `.local-artifacts/issue-15-confirmatory-v2-final`
- exact validation passed:

      python -u -m scripts.run_issue_15_confirmatory_v2 --validate

- no-write dry-run and 27 focused ingestion/integrated/confirmatory tests passed; `git diff --check` passed; `env` is synchronized with `origin/env`.

#16 may now proceed against this frozen capacity-15 oracle-symbol system. Learned-parser degradation must be reported relative to this fixed oracle reference and cannot change or rescue the issue-15 disposition.
